jetstream/probe_sets.py

```diff
@@ -114,20 +114,19 @@ def to_summaries(self, feature_slug: str) -> List[statistics.Summary]:
         return [ever_used, used_mean, used_deciles]
 
 
-FeatureTelemetryType = Union[FeatureEventTelemetry, FeatureScalarTelemetry]
+ProbeType = Union[TelemetryEventProbe, TelemetryScalarProbe]
 
 
 @attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
-class Feature:
+class ProbeSet:
     slug: str
     name: str
-    description: str
```

Comment:
oh, i guess we don't need to plumb this back through to experimenter anymore, so that's nice